docs/faq.rst

```diff
@@ -97,29 +96,27 @@ Computations involving uncommitted data are performed on the default
 device and the results are uncommitted on the default device.
 
 Data can also be placed explicitly on a device using :func:`jax.device_put`
-with a ``device`` parameter, in which case if becomes **committed** to the device:
+with a ``device`` parameter, in which case it—the data—becomes **committed** to the device:
 
 >>> import jax
 >>> from jax import device_put
 >>> print(device_put(1, jax.devices()[2]).device_buffer.device())  # doctest: +SKIP
 gpu:2
 
-Computations involving some committed inputs, will happen on the
-committed device, and the result will be committed on the
-same device. It is an error to invoke an operation on
-arguments that are committed to more than one device.
+Computations involving some committed inputs will happen on the
+committed device and the result will be committed on the
+same device. You should not invoke an operation on
+arguments that are committed to more than one device—it will lead to errors.
 
-You can also use :func:`jax.device_put` without a ``device`` parameter,
-in which case the data is left as is if already on a device (whether
-committed or not), or a Python value that is not on any device is
-placed uncommitted on the default device.
+You can also use :func:`jax.device_put` without a ``device`` parameter. In this case
+the data is left as is if it's already on the device (committed or not), or a Python 
+value—that is not on any device—is placed uncommitted on the default device.
```

Comment:
How about:

You can also use :func:`jax.device_put` without a ``device`` parameter. If the data is already on a device (committed or not), it's left as-is. If the data isn't on any device (i.e. it's a regular Python or numpy value), it's placed uncommitted on the default device.

Reply:
⚡️